Sources/WalletConnectVerify/AttestKeyGenerator.swift

```diff
@@ -21,8 +20,9 @@ class AttestKeyGenerator: AttestKeyGenerating {
     }
 
     func generateKeys() async throws -> String {
-        let keyId = try await service.generateKey()
-        keyIdStorage.set(keyId, forKey: Constants.keyIdStorageKey)
-        return keyId
+        // TODO - Add DCAppAttestService handling for iOS 14.0
+        // let keyId = try await service.generateKey()
+        keyIdStorage.set("keyId", forKey: Constants.keyIdStorageKey)
+        return "keyId"
```

Comment:
what does it change? this class is used ably by the dapp, which we don't support natively now.